project/Dependencies.scala

```diff
@@ -41,7 +41,7 @@ object Dependencies {
     )
   )
 
-  val CirceVersion = "0.14.1"
+  val CirceVersion = "0.14.+"
```

Comment:
can you revert these version changes?  I prefer reproducible builds, and I have seen scenarios where authors have published breaking changes under a minor release.  

When your change comes in, I'll refresh all the libraries myself and publish a new version of slack4s

Reply:
sure thing :+1: